test/e2e/e2e_test.go

```diff
@@ -0,0 +1,39 @@
+package e2e
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/onsi/ginkgo/v2"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/klog/v2"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/framework/config"
+
+	// test sources
+	_ "github.com/kubernetes-csi/external-provisioner/test/e2e/storage"
+)
+
+func TestMain(m *testing.M) {
+	klog.InitFlags(nil)
+	utilruntime.Must(flag.Set("logtostderr", "false"))
+	utilruntime.Must(flag.Set("alsologtostderr", "false"))
+	utilruntime.Must(flag.Set("one_output", "true"))
+	klog.SetOutput(ginkgo.GinkgoWriter)
+
+	config.CopyFlags(config.Flags, flag.CommandLine)
+	framework.RegisterCommonFlags(flag.CommandLine)
+	framework.RegisterClusterFlags(flag.CommandLine)
+	flag.Parse()
+
+	// Register framework flags, then handle flags.
```

Comment:
Nit: this comments belongs before `config.CopyFlags`.

Reply:
Done!